README.md

```diff
@@ -639,6 +645,51 @@ integrator, e.g.:
 </section>
 ```
 
+#### Providing Config Objects
+
+The integrator can provide config objects for Feature Services and Feature Apps
+via the Feature Service registry and Feature App manager:
```

Comment:
I just noticed: we should probably explicitly mention that the keys are the feature app/service IDs.